core/reactor_common.c

```diff
@@ -531,13 +531,13 @@ lf_token_t* create_token(size_t element_size) {
  *  field pointing to newly allocated memory.
  */
 lf_token_t* _lf_initialize_token_with_value(lf_token_t* token, void* value, size_t length) {
-    // assert(token != NULL);
+    assert(token != NULL);
```

Comment:
I think the default CMake build type ("Release") disables asserts. Would that be okay here?

Reply:
I think this is fine. I checked all the callers, and none has the possibility passing a null argument.